statsmodels/tsa/automatic/sarimax.py

```diff
@@ -3,24 +3,92 @@
 import pandas as pd
 import statsmodels.api as sm
 
-def auto_order(endog, criteria='aic', d=0, max_order=(3, 3), **spec=None):
+
+def auto_order(
+                endog, criteria='aic', d=0, max_order=(3, 3),
```

Comment:
The first line arguments should be on the same line as the `def` part.

Reply:
I followed the E127 convention along with the pycodestyle suggestions, so had to take it to the new line.
I'll make the desired changes.